src/widgets/settingspages/HighlightingPage.cpp

```diff
@@ -38,6 +38,8 @@ namespace {
         {"Moderator", "moderator"},
         {"Verified", "partner"},
         {"VIP", "vip"},
+        {"Predicted One", "predictions/blue-1,predictions/pink-1"},
+        {"Predicted Two", "predictions/pink-2,predictions/blue-2"},
```

Comment:
Correct me if I'm wrong, but I believe that `Predicted pink` and `Predicted blue` would fit a bit better in here, As those are usually differed by colors in twitch's frontend.

Reply:
This was one thing that I was confused about, as I had never seen Pink 1 or Blue 2 in predictions before. Just keeping Blue 1 and Pink 2 is probably all that's needed, but I wanted to cover the possible edge case. I can combine them.